src/services/latex.js

```diff
@@ -27,6 +27,12 @@ var latexMathParser = (function() {
   var variable = letter.map(function(c) { return Letter(c); });
   var symbol = regex(/^[^${}\\_^]/).map(function(c) { return VanillaSymbol(c); });
 
+  // Compound commands e.g. \not\subset
+  var compound = regex(/^\\not\\[a-z]+/).then(function(cmp) {
```

Comment:
@eoghanmcilwaine so for now, the only compound commands we want to support are those beginning with `\\not`